frontend/packages/ceph-storage-plugin/src/queries/odf-queries.ts

```diff
@@ -0,0 +1,15 @@
+export enum ODFQueries {
+  LATENCY = 'LAT',
```

Comment:
```suggestion
  LATENCY = 'LATENCY',
```

Reply:
It's more for type checking. The value on the right side of enum really doesn't matter much..

Reply:
still looks wierd